.gitlab-ci.yml

```
@@ -92,6 +92,11 @@ publish:s3:latest:
   only:
     variables:
       - $MENDER_VERSION =~ /[0-9]+\.[0-9]+\.[0-9]+/
-  before_script:
-    - apt update && apt install -yyq awscli
-    - deb_version=latest
+  script:
+    - echo "Publishing latest packages to S3"
+    - for arch in amd64 arm64 armhf; do
+        aws s3 cp output/mender-client_${deb_version}_${arch}.deb
+          s3://${S3_BUCKET_NAME}/latest/${S3_BUCKET_SUBPATH}/${arch}/mender-client_latest_${arch}.deb;
+        aws s3api put-object-acl --acl public-read --bucket ${S3_BUCKET_NAME}
+          --key ${MENDER_VERSION}/${S3_BUCKET_SUBPATH}/${arch}/mender-client_${deb_version}_${arch}.deb;
```

The ACL needs to be applied to the same path as the file:

```suggestion
          --key latest/${S3_BUCKET_SUBPATH}/${arch}/mender-client_latest_${arch}.deb;
```